### ISSUE TITLE
potential fonts module issue

### ISSUE THREAD

**Issue by abathur** [LnL7/nix-darwin#120]
I'm posting a little tentatively on the off chance someone else has insight that'll save me some debug time this evening. I was putzing around with the fonts module to test it out and copied in my font list from nixos (which included a handful of individual fonts, mostly code/mono, powerline, corefonts and vista-fonts). 

It looked like I had this working, but a bit later I noticed Arial showing up on a few sites where it wasn't before (google.com, stack overflow). I haven't successfully fixed or debugged it yet, but I can imagine at least a few ways it could go awry. Example:

<img width="660" alt="screen shot 2019-01-13 at 5 35 43 pm" src="https://user-images.githubusercontent.com/2548365/51092143-b3f1b080-1759-11e9-940f-2c13fdcc280c.png">

**Comment by abathur** [comment]
I have an older macbook at home that is also on Mojave; I compared the /Library/Font directories just now and see what is causing the trouble. The one having trouble was missing (among a few others) a plain `Arial.ttf` that was present on the other system, so I guess Safari was substituting `Arial Black.ttf` in its place.

I copied this over and it started working fine again.

Reading around a bit turned up some suggestions that fontrestore is better at getting rid of new fonts than it is at restoring missing ones, so I suspect that corefonts' arial got linked over the existing copy, and then fontrestore removed the unrecognized copy on a subsequent rebuild.
